e2e/interchaintestv8/relayer_test.go

```diff
@@ -91,3 +104,133 @@ func (s *RelayerTestSuite) TestRelayerInfo() {
 		s.Require().Equal(eth.ChainID.String(), info.TargetChain.ChainId)
 	})
 }
+
+func (s *RelayerTestSuite) TestRelayToEth() {
+	ctx := context.Background()
+	s.SetupSuite(ctx, operator.ProofTypeGroth16)
+
+	eth, simd := s.ChainA, s.ChainB
+
+	ics26Address := ethcommon.HexToAddress(s.contractAddresses.Ics26Router)
+	ics20Address := ethcommon.HexToAddress(s.contractAddresses.Ics20Transfer)
+	transferAmount := big.NewInt(testvalues.TransferAmount)
+	ethereumUserAddress := crypto.PubkeyToAddress(s.key.PublicKey)
+	cosmosUserWallet := s.UserB
+	cosmosUserAddress := cosmosUserWallet.FormattedAddress()
+	sendMemo := "nonnativesend"
+
+	var (
+		transferCoin sdk.Coin
+		txHash       []byte
+	)
+	s.Require().True(s.Run("Send transfer on Cosmos chain", func() {
+		// We need the timeout to be a whole number of seconds to be received by eth
```

Comment:
Nit: I think maybe this comment comes from the other tests and is a relic from when the timeouts were not the same. I think we can just delete this comment everywhere as it is more confusing than clarifying at this point :D

Reply:
done